src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -1280,6 +1283,72 @@ def validate_custom_error_pages(namespace):
     namespace.custom_error_pages = values
 
 
+def validate_custom_headers(namespace):
+
+    if not namespace.monitor_custom_headers:
+        return
+
+    values = []
+    for item in namespace.monitor_custom_headers:
+        try:
+            item_split = item.split('=', 1)
+            values.append({'name': item_split[0], 'value': item_split[1]})
+        except IndexError:
+            raise CLIError('usage error: --custom-headers KEY=VALUE')
+
+    namespace.monitor_custom_headers = values
+
+
+def validate_status_code_ranges(namespace):
+
+    if not namespace.status_code_ranges:
+        return
+
+    values = []
+    for item in namespace.status_code_ranges:
+        item_split = item.split('-', 1)
+        usage_error = CLIError('usage error: --status-code-ranges VAL | --status-code-ranges MIN-MAX')
+        try:
+            if len(item_split) == 1:
+                values.append({'min': int(item_split[0]), 'max': int(item_split[0])})
+            elif len(item_split) == 2:
+                values.append({'min': int(item_split[0]), 'max': int(item_split[1])})
+            else:
+                raise usage_error
+        except ValueError:
+            raise usage_error
+
+    namespace.status_code_ranges = values
+
+
+def validate_subnet_ranges(namespace):
+
+    if not namespace.subnets:
+        return
+
+    values = []
+    for item in namespace.subnets:
+        try:
+            item_split = item.split('-', 1)
```

Comment:
Unlike above, I think subnet strings are constrained enough that I don't think it's a problem to just use a split.